tooling/minimize/tests/pass/slice.rs

```diff
@@ -28,4 +28,5 @@ fn main() {
 
     assert_some_elements(slice);
     assert!(slice.len() == 5);
+    assert!(slice.iter().count() == 5);
```

Comment:
Does actually iterating with a for loop work, too? :)

Reply:
Yessss